### ISSUE TITLE
Inputing a Trajectory Geometry Issues

### ISSUE THREAD

**Issue by Gugym** [MetOffice/qgis-plugin-edr#11]
QGIS3.28.6, Plugin version 0.13.1.

When trying to add a linestring for a trajectory it appears very difficult to provide a user drawn trajectory.

Steps to reproduce:
Open the QGIS Plugin and go to service: https://ogcie.iblsoft.com/edr, select collection GFS - Layer between two depths below land surface and an instance of the data before selecting the data query of trajectory. Click on set spatial extent, then linestring: <NOT SET> to digitise the trajectory path. 

The user appears to be prompted to enter the line but it does not get captured with the coordinates added to the table as expected and  a WKT linestring is not generated from user input. The user is left on the map and has to re-open the EDR dialogue.

**Comment by JanCaha** [comment]
This is linked to #18.

The tool is not designed to draw a line but to select a line from existing layer. The tool allows you to select any line from any layer that contains Linestrings. The line will be highlighted and upon a click transferred into the window with the table. This allows for repeated queries with the same geometry. If user would draw a geometry every time it is unlikely that the drawn geometries would be the same. I discussed this with Matt in one the meetings and we decided upon this approach.

It would be possible to also include tool for line drawing but that would require some additional work.
